### ISSUE TITLE
Restore Focus Outline

### ISSUE THREAD

**Issue by gtarnoff** [ghinda/css-toggle-switch#48]
Currently the demo doesn't have anything to indicate that the toggle switch currently has keyboard focus. While the demo is great in that it can be used by a keyboard and can register to screenreaders, people who have low vision using a keyboard do not know that the item is actionable.

**Comment by ghinda** [comment]
It does have a very subtle gray outline around the switch, but you're right that it's not very visible.
https://github.com/ghinda/css-toggle-switch/blob/master/src/core/shared.scss#L32

Any suggestions?

**Comment by gtarnoff** [comment]
Ideally we'd use the default system. However, since the focus is being put on the `a` for what seems like functionality, that doesn't seem to work so we need to specify something. From the Chrome POV, it would be `outline: -webkit-focus-ring-color auto 5px;`. I currently don't have all the browsers on my machine, but they should have similar attributes you could set.

**Comment by ghinda** [comment]
Fixed with `4.0.2`. I replaced the previous focus outline with
https://github.com/ghinda/css-toggle-switch/blob/master/src/core/shared.scss#L28-L43

It matches the default focus styles as much as possible now.
